src/Execute/ExecutionEnv.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Command} from '../Backend/Command';
+
+interface ExecutionEnv {
+  name(): string;
+  envInfo(): string;
+  runCommand(command: Command): Command;
```

Comment:
Is the role of this function running `command` param?
Then, should return type be some result (e.g., success, fail) after running command ?

Reply:
Oh, I made a mistake. I thought Env just make a command to run, not running command. I will fix it.

I thought `Executor` make `Command` for inference or profile, and `ExecutionEnv` wrap this with bridge command.
I will change a name of function. sorry for confusing.

Reply:
Aha. I understand now.
Then how about something like `wrapCommand()`?

Maybe some comment would help people.

Reply:
thank! I will fix it!